crates/turbo-tasks/src/graph/graph_store.rs

```diff
@@ -66,9 +90,9 @@ where
     }
 }
```

Comment:
`SkipDuplicates` could implement `skip_duplicates` and return self

Reply:
For now, with the way the traits are set up, this would require specialization, so I'm going to skip this for now.